docs/ai/quickstarts/evaluate-ai-response.md

```diff
@@ -10,6 +10,9 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
 
 In this quickstart, you create an MSTest app to evaluate the chat response of a model. The test app uses the [Microsoft.Extensions.AI.Evaluation](https://www.nuget.org/packages/Microsoft.Extensions.AI.Evaluation) libraries.
 
+> [!NOTE]
+> This quickstart demonstrates the simplest usage of the evaluation API. Notably, it doesn't demonstrate use of the [response caching](../conceptual/evaluation-libraries.md#cached-responses) and [reporting](../conceptual/evaluation-libraries.md#reporting) functionality, which are important if you're authoring unit tests that run as part of an "offline" evaluation pipeline. The scenario shown in this quickstart is suitable in use cases such as "online" evaluation of AI responses within production code and logging scores to telemetry, where caching and reporting aren't relevant. For a tutorial that demonstrates the caching and reporting functionality, see [Tutorial: Evaluate a model's response with response caching and reporting](../tutorials/evaluate-with-reporting.md)
+
 ## Prerequisites
 
 - [Install .NET 8.0](https://dotnet.microsoft.com/download) or a later version
```

Comment:
Can we update the quick start to use an Azure Open AI model as well? At the moment, the evaluator prompts generally don't perform too great against local models (even though it may work for one off cases). So, if we show ollama here, we may end up setting the wrong expectation...

Reply:
I opened a separate issue for this.